src/dosext/mfs/lfn.c

```diff
@@ -1328,6 +1328,11 @@ int mfs_lfn(void)
 {
 	int carry, ret;
 
+	if (!mach_fs_enabled) {
```

Comment:
But where is it set to TRUE?
In another patch, as usual? :)
The logic can't be split around
patches, it is not possible to
review this way.